x-pack/plugins/enterprise_search/server/routes/app_search/engines.test.ts

```diff
@@ -110,6 +110,48 @@ describe('engine routes', () => {
     });
   });
 
+  describe('POST /api/app_search/engines', () => {
+    let mockRouter: MockRouter;
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+      mockRouter = new MockRouter({
+        method: 'post',
+        path: '/api/app_search/engines',
+        payload: 'body',
```

Comment:
Just FYI, you'll need to update once Constance's PR is merged.

Reply:
Thanks for this callout! I wasn't seeing this flagged locally when I was running the typescript check, I'll have to look into that